underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystemUtils.java

```diff
@@ -31,11 +31,9 @@ private HdfsUnderFileSystemUtils() {} // prevent instantiation
    * @param hadoopConf configuration to replace the key in
    * @param key the key to replace
    */
-  public static void addKey(org.apache.hadoop.conf.Configuration hadoopConf, String key) {
-    if (System.getProperty(key) != null) {
-      hadoopConf.set(key, System.getProperty(key));
-    } else if (Configuration.get(key) != null) {
-      hadoopConf.set(key, Configuration.get(key));
+  public static void addKey(org.apache.hadoop.conf.Configuration hadoopConf, PropertyKey key) {
```

Comment:
are we sure this is only used to set Alluxio properties? If so we should update the javadoc about the limitation